frontend/packages/console-plugin-sdk/src/codegen/plugin-resolver.ts

```diff
@@ -88,11 +105,14 @@ export const resolvePluginPackages = (
   return pluginFilter(appPackage, pluginPackages);
 };
 
-export type Package = readPkg.NormalizedPackageJson;
+export type Package = readPkg.NormalizedPackageJson & {
+  _path: string;
```

Comment:
underscore dangle :(

Reply:
@christianvogt , hi, 
we discussed this in https://github.com/openshift/console/pull/2051#discussion_r331145854 :-)

will be happy to revert @vojtechszocs ?

Reply:
No need to revert. I'm good with this.

Reply:
> underscore dangle :(

This one is intentional, please see https://github.com/openshift/console/pull/2051#discussion_r340287638